nikola/plugins/command/import_wordpress.py

```diff
@@ -931,14 +954,32 @@ def import_postpage_item(self, item, wordpress_namespace, out_folder=None, attac
                     meta_slug = slug
                 tags, other_meta = self._create_metadata(status, excerpt, tags, categories,
                                                          post_name=os.path.join(out_folder, slug))
-                self.write_metadata(os.path.join(self.output_folder, out_folder,
-                                                 out_meta_filename),
-                                    title, meta_slug, post_date, description, tags, **other_meta)
-                self.write_content(
-                    os.path.join(self.output_folder,
-                                 out_folder, out_content_filename),
-                    content,
-                    rewrite_html)
+
+                meta = {
+                    "title": title,
+                    "slug": meta_slug,
+                    "date": post_date,
+                    "description": description,
+                    "tags": ','.join(tags),
+                }
+                meta.update(other_meta)
+                if self.onefile:
```

Comment:
Why not pass `self.onefile` as another argument to `write_post`, so that all importer plugins can use `write_post` for both single and non-single file output? Now all importer plugins which want to support both ways have to do such a rather complex `if`-`then`-`else` statement.